detail.html

```diff
@@ -0,0 +1,34 @@
+<%inherit file="app_base.htm" />
+
+<%block name ="content_top">
+</%block>
+
+<div id = "pagestuff">
+  <%block name = "content_middle">
+    <div id = "small_imgs">
+    <ul>
+      %for pic in product.image_urls():
+        <li class = "hover_item"><img src = "${pic}" alt = "main picture for ${product_name}"/></li>
+      %endfor
+    </ul>
+    </div>
+    <div id = "big_img">
+      <img src="${product.image_url()}" alt = "alternate picture for ${product_name}"/>
+
+    </div>
+    <div id = "detail_desc">
+    <h1>${product_name}</h1>
+
+      <p class = "prod_desc">${product.description}</p>
+    </div>
+
+    %if product.TITLE == "Bulk":
+      <form id = "q_form">
+        Quantity:
+        <input class="form-control quantityform" type = "number" value="1" name = "quantity" min = "1">
```

Comment:
Good job setting default as 1 and min as 1.

Reply:
Thanks!!